.github/workflows/close-org-prs.yml

```diff
@@ -0,0 +1,51 @@
+name: Close PRs from Orgs
+
+on:
+  pull_request_target:
+    types: [opened, reopened, synchronize]
+
+permissions:
+  pull-requests: write
+  issues: write
```

Comment:
I don't think `issues: write` is needed?

Reply:
Let me try without it after the changes since we removed a couple of stuff.

Reply:
Never mind, creating a comment requires it.